src/pokedex_plus_hgss.c

```diff
@@ -6571,7 +6571,8 @@ static void PrintEvolutionTargetSpeciesAndMethod(u8 taskId, u16 species, u8 dept
                 StringAppend(gStringVar4, COMPOUND_STRING(", "));
             }
 
-            switch((enum EvolutionConditions)evolutions[i].params[j].condition)
+            enum EvolutionConditions condition = evolutions[i].params[j].condition;
+            switch(condition)
```

Comment:
If you're going to save it to a variable, can you also update the rest of the code to use the variable instead?